nikola/utils.py

```diff
@@ -1942,3 +1942,20 @@ def load_data(path):
         return
     with io.open(path, 'r', encoding='utf8') as inf:
         return loader.load(inf)
+
+
+# http://stackoverflow.com/a/2087433
+if sys.version_info[0] == 3 and sys.version_info[1] >= 4:
```

Comment:
Breaks with Python 4.0.

I’d suggest:

```py
try:
    import html
    html_unescape = html.unescape
except (AttributeError, ImportError):
    try:
        # the whole ordeal
```

Or even don’t use html.unescape and always define our own.